en/docs/learn/design-api/create-api/create-a-rest-api-from-a-swagger-definition.md

```diff
@@ -14,8 +14,21 @@ Follow the instructions below to create a REST API using a Open API definition f
 2.  Go to **CREATE API** and Click **I Have an Existing API**.
     ![]({{base_path}}/assets/img/learn/create-a-rest-api-using-existing-swagger-def.jpg)
 
-3.  Select **OpenAPI URL** and provide `http://petstore.swagger.io/v2/swagger.json` as the URL. Click **NEXT** .
-    ![]({{base_path}}/assets/img/learn/create-rest-api-using-swagger-def-form1.jpg)
+3. Select one of the following options:
+
+     * OpenAPI URL - If you select this option, you need to provide an endpoint URL.
+
+     * OpenAPI Archive/File - If you select this option, click Browse and upload either an individual WSDL file or a WSDL archive, which is a WSDL project that has multiple WSDLs.
```

Comment:
individual Open API definition or an archive with Open API definition with external file referencs - link the external ref spec here
_

Reply:
https://github.com/wso2/docs-apim/pull/1178/commits/b2cf8d94b88e8ece796f98d7cf35d513195c3528